### ISSUE TITLE
'for' attributes for labels

### ISSUE THREAD

**Issue by jackplug** [mdo/wtf-forms#69]
Firstly - really nice work, particularly the cross-browser stuff!

Regarding your point about the browser automatically associating the label implicitly with the inner input element, this is true, and for screenreaders, it **should** also be true. Unfortunately (sorry, I cannot locate the testing on this, aside from this on [MDN](https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Forms/How_to_structure_an_HTML_form#The_<label>_element)), but apparently, some screenreaders have a problem dealing with implicit association. Therefore, I would suggest including the `for `attribute in order to 'play safe' - it won't hurt anything elsewhere, as far as I can see

**Comment by StommePoes** [comment]
Dragon NaturallySpeaking.
